azure-pipelines.yml

```diff
@@ -171,6 +171,12 @@ steps:
     platform: '$(buildPlatformNative)'
     configuration: '$(buildConfiguration)'
 
+- task: CopyFiles@2
+  inputs:
+    sourceFolder: '$(sample3PublishedArtifactPath)'
+    contents: '**\$(sample3PublishedArtifactPath)\**\?(*.exe|*.dll|*.pdb)'
+    targetFolder: '$(sample3PublishedArtifactPath)'
+
```

Comment:
Do not merge until resolved. .xml and .txt files are still persisting in the artifacts folder.

Reply:
Is it because the SRC and DST folders are the same? Maybe you need to copy from one frolder to another one

Reply:
Used deletefiles task instead